### ISSUE TITLE
Build is flagged as 'Unsafe' by one of the engines on VirusTotal

### ISSUE THREAD

**Issue by Wufflez** [Wufflez/Loki#1]
No idea how to find more specific details yet, but Cylance appears to be some kind of AI based scanner. Hopefully will figure out what heuristic is causing it to be flagged so it can be worked around.

[VirusTotal report for Loki.exe (0.4 beta)](https://www.virustotal.com/gui/file/6b02cd7c0cca33ccaca657f60d71f7eee2f0207466873ab2f8aaa8500da40686/detection)

**Comment by Wufflez** [comment]
Doing a bit of further digging it seems it doesn't like any .NET Core 3, 3.1 or .NET 5 executables. Even a simple 'Hello world' application is flagged by Cylance. That makes some sense as the logic for the application is all in Loki.dll and not in the executable, which other than a custom icon and some metadata doesn't actually differ very much from the hello world one.

**Comment by kirillovmr** [comment]
I believe that there is no real need to deal with VirusTotal report since the project is open-source, so anyone can verify the integrity of the program.

**Comment by Wufflez** [comment]
The main reason for this being an issue is that many people obviously cannot vet code so easily, especially as the codebase grows. Some sites too I uploaded this mod to such as Nexusmods.com automatically flag uploads and prevent downloads if even 1 virus engine flags it as positive.

I actually spoke to a moderator and got the last upload cleared there, but obviously it's a pain to request manual validation on every upload there.

A quick workaround would be to revert to .NET Framework but it feels wrong, and the language support for .NET Core and .NET 5 (and soon 6) is much better and I've already got quite used to using C# 9 features.

I'm hoping Cylance will soon fix this false positive, it's quite insane that any .NET Core 3.1 or .NET 5 application is automatically flagged as dangerous because of the way the executable is just a bootstrapper for the runtime.
